TODO_MOVE_TO_EDIT_MODE_RFC.md

```diff
@@ -0,0 +1,47 @@
+## Methods
+
+These methods can be overridden in a derived class in order to customize event handling.
+
+### `onLayerClick`
+
+The pointer went down and up without dragging. This method is called regardless if something was picked.
+
+#### `event` argument
+
+* `picks` (Array): An array containing [deck.gl Picking Info Objects](https://uber.github.io/deck.gl/#/documentation/developer-guide/adding-interactivity?section=what-can-be-picked-) for all objects that were under the pointer when clicked, or an empty array if nothing from this layer was under the pointer.
+* `screenCoords` (Array): `[x, y]` screen pixel coordinates relative to the deck.gl canvas.
+* `groundCoords` (Array): `[lng, lat]` ground coordinates.
+
+### `onStartDragging`
+
+The pointer went down on something rendered by this layer and the pointer started to move.
+
+* `picks` (Array): An array containing [deck.gl Picking Info Objects](https://uber.github.io/deck.gl/#/documentation/developer-guide/adding-interactivity?section=what-can-be-picked-) for all objects that were under the pointer when it went down.
+* `screenCoords` (Array): `[x, y]` screen pixel coordinates relative to the deck.gl canvas where the pointer was when it was considered to start dragging (should be very close to `pointerDownScreenCoords`).
+* `groundCoords` (Array): `[lng, lat]` ground coordinates where the pointer was when it was considered to start dragging (should be very close to `pointerDownGroundCoords`).
+* `pointerDownScreenCoords` (Array): `[x, y]` screen pixel coordinates relative to the deck.gl canvas where the pointer went down.
+* `pointerDownGroundCoords` (Array): `[lng, lat]` ground coordinates where the pointer went down.
```

Comment:
`pointerDownGroundCoords -> pointerDownMapCoords` or mentioning it would be replaced by `pointerDownMapCoords`?

Reply:
Full `EditMode` documentation will be done in separate PR